draft-ietf-httpbis-digest-headers.md

```diff
@@ -803,6 +807,8 @@ Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=, id-sha-256=X48E9qO
 iwiAeyJoZWxsbyI6ICJ3b3JsZCJ9Aw==
 ~~~
 
+The above payload body contains non-printable characters and it is shown as base64 encoded.
```

Comment:
same as above comment, lets add this to the response description at the start of section. e.g.

```
The response includes a `Content-Encoding: br` that indicates the selected representation is brotli encoded, the payload body is displayed as base64. There are two `Digest` field-values:

<bullets as they are> 
```